pkg/database/migrations.go

```diff
@@ -1827,6 +1828,21 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return nil
 			},
 		},
+		{
+			ID: "00075-EnableExtension_hstore",
+			Migrate: func(tx *gorm.DB) error {
+				return tx.Exec("CREATE EXTENSION IF NOT EXISTS hstore").Error
+			},
+		},
+		{
+			ID: "00076-AddAlternateSMSTemplates",
```

Comment:
77

and we should really write a test that migrations are incremental... (https://github.com/google/exposure-notifications-verification-server/issues/1343)